packages/engine/errors/1016.md

```diff
@@ -0,0 +1,17 @@
+---
+original: 'A required parameter cannot follow an optional parameter.'
+excerpt: 'Your first parameter cannot be an optional parameter'
```

Comment:
Maybe `'Optional parameters should always come last'`?
although the original is quite concise